frontend/common/src/lang/fr.json

```diff
@@ -1979,5 +1971,23 @@
   "hWHT65": {
     "defaultMessage": "Pour cette procédure de candidature, vous devez remplir les conditions linguistiques suivantes :",
     "description": "Text that appears when a user is missing a language requirement on their profile."
+  },
+  "nLW9zq": {
+    "defaultMessage": "Technologie de l’information"
+  },
+  "6svHxg": {
+    "defaultMessage": "Services administratifs"
+  },
+  "1YuZjR": {
+    "defaultMessage": "Emplois en TI (en cours)",
```

Comment:
Should have new tag?

Reply:
No some of the strings are for select (or multi-select) options which shouldn't be wrapped in abbr tags since the option tag only accepts a string as the child. There should be an aria label though! Also, I haven't touched any of the multiselect options in this issue.